src/sentry/static/sentry/app/styles/global.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Global, css} from '@emotion/core';
+
+import {Theme} from 'app/utils/theme';
+
+const styles = (theme: Theme) => css`
```

Comment:
I would prefer to use the `styled(Global)` approach, but I believe it doesn't work atm, right?

Reply:
Is that a thing we're supposed to be able to do?